src/App.vue

```diff
@@ -251,14 +252,12 @@ export default {
         return require('@/graphql/Agent/agents.js').default(this.isCloud)
       },
       pollInterval: 3000,
-      skip() {
-        return (
-          (this.isCloud && (!this.isAuthenticated || !this.isAuthorized)) ||
-          !this.connected ||
-          (this.isServer && !this.connected)
-        )
-      },
-      update: data => data.agent
+      //Without this, server UI with no actual server shows results
+      fetchPolicy: 'no-cache',
+      update({ agent }) {
+        if (!agent) return null
+        return agent
+      }
```

Comment:
I'm not sure I understand why this change was made, unless we need `null` specifically, in which case we could do:
```js
update: data => data.agent ?? null
```

Reply:
Yup wanted to be sure we had null.  But I like your suggestion better - I'll update.